package-lock.json

```diff
@@ -1,8 +1,6454 @@
 {
   "name": "download.jqueryui.com",
   "version": "2.1.22",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This was generated using npm 7 on Node 14, right? In that case, please also update `.nvmrc` too say `14` instead of `10	.

Reply:
Thanks! Will push this shortly.